richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/paragraph/type/OrderedList.kt

```diff
@@ -11,6 +11,7 @@ import com.mohamedrejeb.richeditor.paragraph.RichParagraph
 
 internal class OrderedList(
     number: Int,
+    private val indent: Int = 38,
```

Comment:
The indent param here shouldn't have a default value. It should be passed by the state instead.